app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/AdyenExceptionMapper.kt

```diff
@@ -0,0 +1,135 @@
+import com.appcoins.payments.methods.adyen.repository.ConflictException
+import com.appcoins.payments.methods.adyen.repository.CurrencyNotSupportedException
+import com.appcoins.payments.methods.adyen.repository.CvcDeclinedException
+import com.appcoins.payments.methods.adyen.repository.CvcLengthException
+import com.appcoins.payments.methods.adyen.repository.DeclinedException
+import com.appcoins.payments.methods.adyen.repository.DeclinedNonGenericException
+import com.appcoins.payments.methods.adyen.repository.ExpiredCardException
+import com.appcoins.payments.methods.adyen.repository.ForbiddenException
+import com.appcoins.payments.methods.adyen.repository.FraudException
+import com.appcoins.payments.methods.adyen.repository.FraudRefusalException
+import com.appcoins.payments.methods.adyen.repository.IncorrectOnlinePinException
+import com.appcoins.payments.methods.adyen.repository.InvalidAmountException
+import com.appcoins.payments.methods.adyen.repository.InvalidCardException
+import com.appcoins.payments.methods.adyen.repository.InvalidCardNumberException
+import com.appcoins.payments.methods.adyen.repository.InvalidCountryCodeException
+import com.appcoins.payments.methods.adyen.repository.IssuerSuspectedFraudException
+import com.appcoins.payments.methods.adyen.repository.IssuerUnavailableException
+import com.appcoins.payments.methods.adyen.repository.MissingBillingAddressException
+import com.appcoins.payments.methods.adyen.repository.Not3dAuthenticatedException
+import com.appcoins.payments.methods.adyen.repository.NotAllowedException
+import com.appcoins.payments.methods.adyen.repository.NotEnoughBalanceException
+import com.appcoins.payments.methods.adyen.repository.NotSupportedException
+import com.appcoins.payments.methods.adyen.repository.OutdatedCardException
+import com.appcoins.payments.methods.adyen.repository.PaymentErrorException
+import com.appcoins.payments.methods.adyen.repository.PaymentNotSupportedException
+import com.appcoins.payments.methods.adyen.repository.PinTriesExceededException
+import com.appcoins.payments.methods.adyen.repository.ReferralException
+import com.appcoins.payments.methods.adyen.repository.RestrictedCardException
+import com.appcoins.payments.methods.adyen.repository.RevocationOfAuthException
+import com.appcoins.payments.methods.adyen.repository.TransactionAmountExceededException
+import com.appcoins.payments.methods.adyen.repository.TransactionNotPermittedException
+import com.appcoins.payments.methods.adyen.repository.WithdrawAmountExceededException
+import com.aptoide.android.aptoidegames.R
+
+fun getAdyenErrorMessage(e: Throwable) = when (e) {
+  /*Generic errors*/
+  is FraudException,
+  is MissingBillingAddressException,
+  is ConflictException,
+  is NotAllowedException,
+  is ForbiddenException,
+  -> null //R.string.error_message_generic_title
+
+  /*Adyen errors*/
+  is InvalidCardException -> R.string.purchase_error_invalid_credit_card_title
```

Comment:
Hardcode the strings on this file

Reply:
It might be better for this to be on hold until the PR with all the strings is merged